Makefile

```diff
@@ -348,6 +348,15 @@ gowork: ## Generate go.work file to support our multi module repository
 	go work use ./api
 	go work sync
 
+PHONY: force-bump
+force-bump: ## Force bump after tagging
+	for dep in $$(cat go.mod | grep  openstack-k8s-operators | grep -vE -- 'indirect|dataplane-operator' | awk '{print $$1}'); do \
```

Comment:
You can use regular expressions in `awk` to make this a bit more concise.  Same goes for the second script.

Reply:
I prefer readability to concision